02-technical-guides/hierarchies/01-hierarchy-fundamentals.ipynb

```diff
@@ -0,0 +1,1605 @@
+{
```

Comment:
<p>if <strong><em>we</em></strong> create <strong><em>our</em></strong> cube in auto or no_measures mode... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Fixed!</p>